src/file-manager/fm-icon-view.c

```diff
@@ -794,6 +801,16 @@ update_layout_menus (FMIconView *view)
         gtk_action_set_label (action, _("_Organize Desktop by Name"));
     }
 
+	action = gtk_action_group_get_action (view->details->icon_action_group,
+                                          FM_ACTION_CLEAN_UP_BY_TIME);
+    gtk_action_set_sensitive (action, !is_auto_layout);
+
+    if (FM_IS_DESKTOP_ICON_VIEW (view))
+    {
+        gtk_action_set_label (action, _("Organize Desktop by Create Time"));
```

Comment:
This is inaccurate, `CAJA_FILE_SORT_BY_MTIME` is for Modified Time, not Created Time.